docs/11-circleci/01-getting-started.mdx

```diff
@@ -0,0 +1,150 @@
+---
+toc_max_heading_level: 4
+---
+
+# Getting started
+
+This guide will help you test and build your Unity project using the
+[Unity Orb](https://circleci.com/developer/orbs/orb/game-ci/unity).
+
+## Overall steps
+
+1. Understand how CircleCI works.
+2. Set up a workflow for your project.
+3. Download the artifacts from your first build using the orb.
+
+## CircleCI
+
+If you are not familiar with CircleCI, read the official documentation on how to set up a
+[workflow](https://circleci.com/docs/getting-started). Subsequent steps will assume you have done
+so.
+
+#### Unity Orb
+
+[Orbs](https://circleci.com/orbs/) are reusable snippets of code that help automate repeated
+processes, speed up project setup and make it easy to integrate with third-party tools.
+
+To streamline the process of testing and building Unity projects in CircleCI, GameCI provides the
+[Unity Orb](https://circleci.com/developer/orbs/orb/game-ci/unity) — a certified orb capable of
+dependencies caching, artifacts storage, test data collection and macOS, Windows, Docker, Machine
+and Runner support.
```

Comment:
```suggestion
To streamline the process of testing and building Unity projects in CircleCI, we provide the
[Unity Orb](https://circleci.com/developer/orbs/orb/game-ci/unity) — a certified orb capable of
caching dependencies, storing artifacts and collecting test data, supporting macOS and Windows with Docker, Machine and Runner options.
```

Reply:
> supporting macOS and Windows with Docker, Machine and Runner options.

This is technically correct, but it doesn't relay what I was trying to say. In CircleCI, "Docker" is a type of executor, which is why it's part of the list in my original sentence and a separate item from Windows.

I'm changing the phrase to:

```mdx
To streamline the process of testing and building Unity projects in CircleCI, we provide the
[Unity Orb](https://circleci.com/developer/orbs/orb/game-ci/unity) — a certified orb capable of
caching dependencies, storing artifacts, collecting test data and running on several 
[executors](/docs/circleci/executors).

```

Let me know what you think!

Reply:
Great!